app/views/partner_requests/index.html.erb

```diff
@@ -15,29 +15,19 @@
           </li>
           <li class="breadcrumb-item">
             <a href="<%= partner_path(current_partner) %>"><%= "#{current_partner.name}" %></a></li>
-          <li class="breadcrumb-item"><a href="#">Diaper Requests</a></li>
+          <li class="breadcrumb-item"><a href="#">Essentials Requests</a></li>
         </ol>
       </div>
     </div>
   </div><!-- /.container-fluid -->
 </section>
-
-
+  <%= render partial: "shared/request_card" %>
```

Comment:
can you also add the `verified` check here as well? ie- ` <%= render "shared/request_card" if @partner.verified? %>`

Reply:
`if @partner.verified?` has been added to line 24. I am not sure if I need to create a new pull request. Thank you for your patience with me.